sdk/deepbook/src/client.ts

```diff
@@ -665,35 +665,71 @@ export class DeepBookClient {
 	 * @param poolId the pool id, eg: 0xcaee8e1c046b58e55196105f1436a2337dcaa0c340a7a8c8baf65e4afb8823a4
 	 * @param lowerPrice lower price you want to query in the level2 book, eg: 18000000000. The number must be an integer float scaled by `FLOAT_SCALING_FACTOR`.
 	 * @param higherPrice higher price you want to query in the level2 book, eg: 20000000000. The number must be an integer float scaled by `FLOAT_SCALING_FACTOR`.
-	 * @param side { 'bid' | 'ask' } bid or ask side
+	 * @param side { 'bid' | 'ask' } bid or ask side. If left undefined, both sides will be returned.
 	 */
 	async getLevel2BookStatus(
 		poolId: string,
 		lowerPrice: bigint,
 		higherPrice: bigint,
-		side: 'bid' | 'ask',
-	): Promise<Level2BookStatusPoint[]> {
+		side: 'bid' | 'ask' | undefined = undefined,
```

Comment:
I really would prefer if this got moved into an enum class and we stop using magic strings here.

Reply:
but for now can we use some string like 'both' and force users to type out the intent